src/map/skill.cpp

```diff
@@ -5107,6 +5107,19 @@ int skill_castend_damage_id (struct block_list* src, struct block_list *bl, uint
 	map_freeblock_lock();
 
 	switch(skill_id) {
+	case HFLI_MOON:	//[orn]
+	case HFLI_SBR44:	//[orn]
+	{
+#ifdef RENEWAL
+		if (hd != nullptr)
+		{
+			skill_blockhomun_start(hd, skill_id, skill_get_cooldown(skill_id, skill_lv));
+			if (hd->master != nullptr)
+				skill_blockpc_start(hd->master, skill_id, skill_get_cooldown(skill_id, skill_lv));
```

Comment:
```suggestion
```
This can be removed with the adjustment to `skill_blockhomun_start()` adjustment.